src/Artsy/Relay/createRelaySSREnvironment.ts

```diff
@@ -79,6 +79,7 @@ export function createRelaySSREnvironment(config: Config = {}) {
       // or further refactoring to extract `addMiddlewareToEnvironment(environment)`,
       // to be used in the SearchBar QueryRenderer (for example).
       next => req => {
+        console.log({ req })
```

Comment:
(extra)

Reply:
haha just beat you to it by 44 seconds 

![image](https://user-images.githubusercontent.com/1242537/55403920-2f901c00-5557-11e9-9ffc-f21c5473f899.png)

Also just fixed it